tests/ui/conftest.py

```diff
@@ -104,8 +103,8 @@ def pytest_configure(config):
     prefix_indexes(config)
 
 
-@pytest.fixture(scope='session')
-def es_test(pytestconfig):
+@pytest.fixture
+def es_test(pytestconfig, db):
```

Comment:
Why did this change? We shouldn't enable database access to all elasticsearch tests by default.